config/uc-r2.yaml

```
@@ -0,0 +1,90 @@
+image:
+  mirrorbits:
+    tag: v0.1.2
+    pullPolicy: IfNotPresent
+  files:
+    # repo & '@sha256' suffix here as we're retrieving a docker digest from updatecli
+    repository: httpd@sha256
+    tag: f70876d78442771406d7245b8d3425e8b0a86891c79811af94fb2e12af0fadeb
+    pullPolicy: IfNotPresent
+
+ingress:
+  enabled: true
+  className: public-nginx
+  annotations:
+    "cert-manager.io/cluster-issuer": "letsencrypt-prod"
+    "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+  hosts:
+    - host: test-hlm-uc-r2
+      paths:
+        - path: /
+          serviceNameSuffix: files
+        - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html)$
+  tls:
+    - secretName: uc-r2-tls
```

```suggestion
    - secretName: azure.updates.jenkins.io
```

If you want the TLS cert to be created, you need an hostname in a DNS zone in which Cert manager is set up to create DNS records